bootstrap-salt.sh

```diff
@@ -2599,7 +2561,7 @@ install_ubuntu_git() {
 install_ubuntu_stable_post() {
     # Workaround for latest LTS packages on latest ubuntu. Normally packages on
     # debian-based systems will automatically start the corresponding daemons
-    if [ "$DISTRO_MAJOR_VERSION" -lt 15 ]; then
+    if [ "$DISTRO_MAJOR_VERSION" -lt 16 ]; then
         return 0
     fi
```

Comment:
Looks like this whole `if` statement is really a legacy of the previous versions. We do not trying to start Salt daemons here in the post install function for Ubuntu, but only enabling them to start on boot. Generally it is done automatically for all packages regardless of distro version, but the later code just double checks that.

I think the above statement could be safely removed.